### ISSUE TITLE
SNP density plot

### ISSUE THREAD

**Issue by deepti100rao** [YinLiLin/CMplot#103]
I have two questions for generating rectangular SNP density plot:

1. Is it necessary to give SNP numbers in the first column? Are serial numbers taken like 1, 2, 3.. or does it have to be SNP1, SNP2...?

2. Is there a limit on the input file size?

**Comment by YinLiLin** [comment]
Yes, the order of columns should be exactly the same with the tutorial data (i.e., id, chr, pos, p). 
There are no mandatory format requirements for the SNP names, any values are acceptable.
Theoretically, no limitation on the file size or the number of SNPs for CMplot, but it is restricted by the memory size of your computer.

**Comment by deepti100rao** [comment]
The package worked seamlessly. Thanks! I am attaching a picture of my plots
side by side using two different references.
Since the colour coding is sort of relative, I am not able to see a
comparison in SNP density between the two references, which is what I want.
Can I get the darkest green for numbers 1000-2000, and so on for example?
Can you help with this?

**Comment by deepti100rao** [comment]
In fact, I must add that the reference genome used in the case of the plot
on the right is closer to the query genome and fewer SNPs were called
against it compared to the one on the left. But, the plot depicts the
opposite. Seeking your help in fine-tuning this. Thanks a lot in advance!

**Comment by YinLiLin** [comment]
Thank you for the feedback. In the latest version, there is a parameter 'bin.breaks=c(min, max, step)' can be used to set the colors, you can set the same 'bin.breaks' for two plots, then it would be comparable on colors between plots.
